build.xml

```diff
@@ -2064,6 +2158,16 @@
     <install pomFile="${build.dir}/${final.name}.pom"
              file="${build.dir}/${final.name}-javadoc.jar"
              classifier="javadoc"/>
+
+    <!-- the cassandra-sstableloader jar -->
+    <install pomFile="${build.dir}/${final.name}-sstableloader.pom"
```

Comment:
Tested this locally and it does appear to work:

```kotlin
plugins {
    id("java")
}

group = "org.example"
version = "1.0-SNAPSHOT"

repositories {
    mavenCentral()
    mavenLocal()
}

dependencies {
    implementation("org.apache.cassandra:cassandra-sstableloader:5.1-SNAPSHOT")
    testImplementation(platform("org.junit:junit-bom:5.10.0"))
    testImplementation("org.junit.jupiter:junit-jupiter")
}

tasks.test {
    useJUnitPlatform()
}
```

![Screenshot From 2025-02-13 19-37-11](https://github.com/user-attachments/assets/ad767c1b-4670-4d63-a215-9654fda4269a)